### PR TITLE
Ztw 2329

### PR DESCRIPTION
**Pull request by jmolnar-zscaler**
simple only so far

### CODE REVIEW

**Review by vkrishnamurthy-zscaler — APPROVED** [review]
looks good.  Changes are more than remote server port.  Better to add another ZTW to indicate these changes, so that tester knows to verify it in addition.

File: cloudformation-templates/zs_cc_cf_template_zscc_macro.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: Zscaler Cloud Connector Pre-Deployment Macro Template
+Description: Zscaler Cloud Connector Pre-Deployment Macro Template v1.3.0
```
Comment (anichari) [inline cloudformation-templates/zs_cc_cf_template_zscc_macro.yaml:2]: If we add versioning in more than one place its just a maintenance hazard. This description is not going to appear anywhere the customer sees. The Outputs version should be the one we use

File: cloudformation-templates/zs_cc_cf_template_simple.yaml [hunk 2]
```diff
@@ -231,21 +245,132 @@ Resources:
...
     Properties:
       VpcId: !Ref VPCID
-      GroupDescription: Enable intra cloud comm
-      SecurityGroupIngress:
-        - IpProtocol: '-1'
-          FromPort: '0'
-          ToPort: '65535'
+      GroupName: cc-svc-sg
+      GroupDescription: Security group for Cloud Connector service interfaces
+  ServiceEgressRule1:
```
Comment (anichari) [inline cloudformation-templates/zs_cc_cf_template_simple.yaml:294]: Can we add better rule names please. Something more descriptive than ServiceEgressRule1

File: cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml [hunk 3]
```diff
@@ -438,37 +451,149 @@ Mappings:
...
     Properties:
       VpcId: !Ref NetworkVpcId
-      GroupDescription: Enable intra cloud comm
-      SecurityGroupEgress:
-        - IpProtocol: '-1'
-          FromPort: '0'
-          ToPort: '0'
+      GroupName: cc-mgmt-sg
+      GroupDescription: Security group for Cloud Connector management interface
+  MgmtEgressRule1:
```
Comment (anichari) [inline cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml:460]: Please let's add better rule names, maybe something like MgmtEgressTcp443Rule ?

File: cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: Zscaler Cloud Connector Auto Scaling Group Template
+Description: Zscaler Cloud Connector Starter Deployment Template with Auto Scaling and GWLB v1.3.0
```
Comment (anichari) [inline cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml:2]: Let's use the version in outputs, versioning in too many places will be a maintenance headache

**Review by anichari — COMMENTED** [review]
Please address comments

File: cloudformation-templates/zs_cc_cf_template_simple.yaml (hunk 2 above)
Comment (jmolnar-zscaler) [reply cloudformation-templates/zs_cc_cf_template_simple.yaml]: @anichari names fixed in https://github.com/zscaler/cloud-native-aws-cloud-connector-deploy/pull/44/commits/d5133a38a05a821c1784e49e3cb596c1cbb772f8

File: cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml (hunk 3 above)
Comment (jmolnar-zscaler) [reply cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml]: https://github.com/zscaler/cloud-native-aws-cloud-connector-deploy/pull/44/commits/d5133a38a05a821c1784e49e3cb596c1cbb772f8

File: cloudformation-templates/zs_cc_cf_template_simple.yaml [hunk 5]
```diff
@@ -36,8 +38,12 @@ Metadata:
         default: Zscaler product
       VPCID:
         default: Choose a VPC ID to Deploy Cloud Connector
-      VPCIpCidrBlock:
-        default: IP CIDR Range for Ingress Security Group Rules
+      ServiceIntSGCidr:
+        default: IP CIDR Range for Service Ingress Security Group Rules
+      SSHInboundSGCidr:
+        default: IP CIDR Range for Management SSH Ingress Security Group Rule
+      ZSSupportServerIP:
```
Comment (rhartnett-zscaler) [inline cloudformation-templates/zs_cc_cf_template_simple.yaml]: How does a customer know this?

File: cloudformation-templates/zs_cc_cf_template_simple.yaml [hunk 6]
```diff
@@ -140,11 +146,18 @@ Parameters:
         AllowedValues:
           - true
           - false
-  VPCIpCidrBlock:
+  ServiceIntSGCidr:
```
Comment (rhartnett-zscaler) [inline cloudformation-templates/zs_cc_cf_template_simple.yaml]: Any migration challenges for customers using the template right now? in the description should we mention that this was previously VPCIpCidrBlock?

File: cloudformation-templates/zs_cc_cf_template_simple.yaml (hunk 5 above)
Comment (rhartnett-zscaler) [reply cloudformation-templates/zs_cc_cf_template_simple.yaml]: Question is answered below, ignore this

**Review by rhartnett-zscaler — APPROVED** [review]

File: cloudformation-templates/zs_cc_cf_template_simple.yaml (hunk 6 above)
Comment (jmolnar-zscaler) [reply cloudformation-templates/zs_cc_cf_template_simple.yaml]: yeah sorry this PR is a bit confusing as i added a couple cidr/ip related parameters and then opted to remove them after further discussion

**Pull request merged**